app/Http/Controllers/AuthController.php

```diff
@@ -8,4 +8,16 @@ class AuthController extends Controller {
     public function loginPage() {
         return inertia('LoginPage');
     }
+    public function registrationPage() {
+        return inertia('RegistrationPage');
+    }
+    public function schedulePage() {
+        return inertia('SchedulePage');
+    }
+    public function courseworkPage() {
+        return inertia('CourseworkPage');
+    }
+    public function import_coursePage() {
+        return inertia('Import_CoursePage');
+    }
```

Comment:
These should be separated into their own controller. And actually while you are at it, let's rename AuthController to just LoginController and we'll have a separate RegisterController.